### ISSUE TITLE
display darknet_no_gpu.exe Stop running!!!help me please

### ISSUE THREAD

**Issue by cuteyuyu** [AlexeyAB/darknet#656]
In accordance with the steps, 
there were originally new training weights and then different data retraining
and...
C:\YOLO\darknet-master\build\darknet\x64>darknet_no_gpu.exe detector train data/obj.data cfg/yolov2-tiny-obj.cfg cfg/yolov2-tiny-voc.conv.13 -dont_show
***** VIDEOINPUT LIBRARY - 0.1995 - TFW07 *****
yolov2-tiny-obj
layer     filters    size              input                output
    0 conv     16  3 x 3 / 1   416 x 416 x   3   ->   416 x 416 x  16
    1 max          2 x 2 / 2   416 x 416 x  16   ->   208 x 208 x  16
    2 conv     32  3 x 3 / 1   208 x 208 x  16   ->   208 x 208 x  32
    3 max          2 x 2 / 2   208 x 208 x  32   ->   104 x 104 x  32
    4 conv     64  3 x 3 / 1   104 x 104 x  32   ->   104 x 104 x  64
    5 max          2 x 2 / 2   104 x 104 x  64   ->    52 x  52 x  64
    6 conv    128  3 x 3 / 1    52 x  52 x  64   ->    52 x  52 x 128
    7 max          2 x 2 / 2    52 x  52 x 128   ->    26 x  26 x 128
    8 conv    256  3 x 3 / 1    26 x  26 x 128   ->    26 x  26 x 256
    9 max          2 x 2 / 2    26 x  26 x 256   ->    13 x  13 x 256
   10 conv    512  3 x 3 / 1    13 x  13 x 256   ->    13 x  13 x 512
   11 max          2 x 2 / 1    13 x  13 x 512   ->    13 x  13 x 512
   12 conv   1024  3 x 3 / 1    13 x  13 x 512   ->    13 x  13 x1024
   13 conv   1024  3 x 3 / 1    13 x  13 x1024   ->    13 x  13 x1024
   14 conv     30  1 x 1 / 1    13 x  13 x1024   ->    13 x  13 x  30
   15 detection
mask_scale: Using default '1.000000'
Loading weights from cfg/yolov2-tiny-voc.conv.13...
 seen 32
Done!
Learning Rate: 0.001, Momentum: 0.9, Decay: 0.0005
Resizing
384
display darknet_no_gpu.exe Stop running!!!help me please

**Comment by AlexeyAB** [comment]
* Your dataset looks like correct.

* Can you do detection using pre-trained weights? https://pjreddie.com/media/files/yolov2-tiny.weights
`darknet_no_gpu.exe detector test data/coco.data cfg/yolov2-tiny.cfg yolov2-tiny.weights dog.jpg`

* Try to change this line: https://github.com/AlexeyAB/darknet/blob/9f2dbfa7df981d41c070261c4b9a8a875b290203/src/gemm.c#L74
to this: `#ifdef NONE`

----

I just ran the same command and it works well:
`darknet_no_gpu.exe detector train data/obj.data tiny-yolo-obj.cfg tiny-yolo-voc.conv.13 -dont_show`
![image](https://user-images.githubusercontent.com/4096485/38883290-1cded3ee-4275-11e8-88d8-39de2be6a0f0.png)

**Comment by cuteyuyu** [comment]
@AlexeyAB 
the first is ok!!
![333](https://user-images.githubusercontent.com/38469710/38884464-45c96e04-42a2-11e8-8320-6253f6495ee0.JPG)

the second 
![444](https://user-images.githubusercontent.com/38469710/38884531-83ceb7f4-42a2-11e8-8b46-265b901d1b4f.JPG)

At last still wrong!!!
![555](https://user-images.githubusercontent.com/38469710/38884725-06e81b8a-42a3-11e8-88ab-588a37ecdf37.JPG)

this is my yolov2-tiny-obj.cfg
[net]
# Testing
batch=16
subdivisions=8
# Training
# batch=64
# subdivisions=2
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
max_batches = 40200
policy=steps
steps=-1,100,20000,30000
scales=.1,10,.1,.1

[convolutional]
batch_normalize=1
filters=16
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=1

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

###########

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=30
activation=linear

[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .6
random=1

**Comment by AlexeyAB** [comment]
Do you use OpenCV and what version?

Try to compile without OpenCV and run training: 
`\darknet.sln` -> (right click on project) -> properties -> C/C++ -> Preprocessor -> Preprocessor Definitions, and remove here: `OPENCV`;

**Comment by cuteyuyu** [comment]
@AlexeyAB 
I use opencv3.1.0 MVS2015 I don't have gup
Try to compile without OpenCV and run training in \darknet.sln or \darknet_no_gpu.sln??

**Comment by AlexeyAB** [comment]
Try to set `random=0` in your cfg-file.

----

> and \darknet.sln is failed to load no project，What happened?

https://github.com/AlexeyAB/darknet#how-to-compile-on-windows

> If you have other version of CUDA (not 9.1) then open build\darknet\darknet.vcxproj by using Notepad, find 2 places with "CUDA 9.1" and change it to your CUDA-version, then do step 1

**Comment by cuteyuyu** [comment]
@AlexeyAB 
- If you don't have GPU, but have MSVS 2015 and OpenCV 3.0 (with paths: C:\opencv_3.0\opencv\build\include & C:\opencv_3.0\opencv\build\x64\vc14\lib), then start MSVS, open build\darknet\darknet_no_gpu.sln, set x64 and Release, and do the: Build -> Build darknet_no_gpu

Is it because of the opencv3.1.0 program can not be implemented?
\darknet_no_gpu.sln remove OPENCV can run it,but no opencv unable to display picture results.
so do I want to change to opencv3.0  solve this problem?
I don't have a gpu so I can't download cuda!
